src/translation/languages/fr.json

```
@@ -0,0 +1,4 @@
+{
+  "test.key": "Clé de test",
+  "test.keyWithReplacedValue": "Pour moi, c’est un {value}. Un {value}, tout le monde sait ce que c’est…"
+}
```

Why don't you do this : 
```suggestion
{
  "test": {
     "key": "Clé de test",
     "keyWithReplacedValue": "Pour moi, c’est un {value}. Un {value}, tout le monde sait ce que c’est…"
  }
}
```

---

Поскольку мы находимся в полностью созданном модуле, который не поддерживает детей, вот почему

---

mais ça peut-être une feature plus tard je suis totalement d'accord, et effectivement cet hiérarchie serait bien meilleur